x-pack/test_serverless/shared/lib/security/kibana_roles/project_controller_security_roles.yml

```diff
@@ -1,6 +1,113 @@
-# -----
-# Source: https://github.com/elastic/project-controller/blob/main/internal/project/security/config/roles.yml
```

Comment:
is this source comment intended to be removed?

Reply:
👍  good catch, it wasn't intentional :)